### DISCUSSION TITLE
How to make the module automatically take effect for the new APP?

### DISCUSSION THREAD

**Discussion by andeaker** [LSPosed/LSPosed#1024] [Q&A] [answered]
After setting up some modules (such as Thanox, XPrivacyLua), if you install a new APP, you must go back to LSPosed, find the corresponding APP in the module and check it to enable it. 

Can LSPosed automatically take effect for the new APP by default?

**Comment by JasonKhew96** [comment, marked as answer]
https://github.com/LSPosed/LSPosed/issues/9

**Reply by andeaker** [reply]
thank you!
